doc/whats_new/v0.21.rst

```diff
@@ -40,6 +40,13 @@ Support for Python 3.4 and below has been officially dropped.
 - An entry goes here
 - An entry goes here
 
+:mod:`sklearn.manifold`
+......................
+
+- |Enhancement| add ``metric_params`` and ``p`` parameters to
+  :class:`manifold.TSNE` constructor for additional parameters of distance
+  metric to use in optimization. :issue:`11793` by :user:`Muneeb Aadil <muneebaadil>`.
```

Comment:
```suggestion
  metric to use in optimization. :issue:`12387` by :user:`Muneeb Aadil <muneebaadil>`.
```
Use pull-request number.

Reply:
Would it make sense to start using :pr: then?